docs/mochitests.md

```diff
@@ -78,6 +78,19 @@ Any of the below APIs that takes a `url` will match it as a substring, meaning t
 * `waitForThreadEvents(dbg, eventName)` - Waits for specific thread events
 * `waitForDispatch(dbg, type)` - Wait for a specific action type to be dispatch. If an async action, will wait for it to be done.
 
+## Pausing
```

Comment:
That's a good point - happy to add it to the API and also add some of the newer methods like clickElement, findelement...